docs/pageserver-tenant-migration.md

```diff
@@ -0,0 +1,20 @@
+## Pageserver tenant migration
+
+### Overview
+
+This feature allows to migrate a timeline from one pageserver to another by utilizing remote storage capability.
+
+### Migration process
+
+Pageserver implements two new http handlers: timeline attach and timeline detach.
+Timeline migration is performed in a following way:
+1. Timeline attach is called on a target pageserver.
+2. For now it is necessary to manually initialize replication stream via callmemaybe call so target pageserver initializes replication from safekeeper (it is desired to avoid this and initialize replication directly in attach handler, but this requires some refactoring TODO create issue)
```

Comment:
> TODO create issue

IMO, issues can be created without getting pushed into repo as TODOs.
There's at least one more similar TODO below that benefits from an actual issue created.

Reply:
Oops, this todo is for me to jut not forget to create an issue :) This particular one is added to the checklist in rebalancing epic, so I'll just remove it here